examples/cms-dotcms/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "private": true,
+  "scripts": {
+    "dev": "next",
+    "build": "next build",
+    "start": "next start"
+  },
+  "dependencies": {
+    "@types/node": "18.0.3",
+    "@types/react": "18.0.15",
+    "classnames": "2.3.1",
+    "cross-fetch": "^3.1.5",
+    "date-fns": "2.25.0",
+    "next": "^12.2.0",
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2",
+    "typescript": "4.7.4"
+  },
+  "devDependencies": {
+    "@tailwindcss/typography": "0.5.2",
+    "autoprefixer": "10.4.2",
+    "postcss": "8.4.5",
+    "tailwindcss": "^3.0.15"
```

Comment:
let's make sure `dependencies` and `devDependencies` are separated correctly, and all dependencies are up-to-date.